README.md

```diff
@@ -563,21 +563,48 @@ default in a reduced functionality ("lite") mode that does not require
 PostgreSQL. To enable the PostgreSQL backend (and the expanded functionality),
 dcrdata may be started with the `--pg` switch.
 
+## APIs
+
+The dcrdata block explorer is exposed by two APIs: a Decred implementation of the [Insight API](https://github.com/bitpay/insight-api) (EXPERIMENTAL), and its own JSON HTTP API. The Insight API uses the path prefix `/insight/api`. The dcrdata API uses the path prefix `/api`.
+
 ### Insight API (EXPERIMENTAL)
 
-The [Insight API](https://github.com/bitpay/insight-api) is accessible via HTTP
-at the `/insight/api` URL path prefix, and via WebSocket at the
-`/insight/socket.io` URL path prefix.
+The [Insight API](https://github.com/bitpay/insight-api) is accessible via HTTP via REST or WebSocket. 
+
+To call the REST API, use the `/insight/api` path prefix. To call the Websocket API, use the `/insight/socket.io` path prefix.
 
 The following endpoints are not implemented: `status`, `sync`, `peer`, `email`,
 and `rates`.
 
+#### Endpoint List
+
+| Method           | Path                   | 
+| -------------------- | ---------------------- | 
+| Block              | `/block/{hash}`          |  
+| Block index          | `/block-index/{height}`      |  
+| Raw block (hash)               | `/rawblock/{hash}`   | 
+| Raw block (height)                | `/rawblock/{height}`     |  
+| Raw summaries               | `/blocks (URL query params: limit, blockDate, ...) `   |              
+| Transaction                 | `/tx/{hash}`     |  
+| Raw transaction              | `/rawtx/{hash}`  | 
+| Address         | `/addr/{address}`       | 
+| Address Props   | `/balance, totalReceived, totalSent, /totalSent, unconfirmedBalance` |  
+| UTXOs | `/addr/{address}/utxo`  |  
+| Multi-address UTXOs | `/addrs/{address0, address1, etc}/utxo`  |  
+| POST-variant of multi-address UTXOs | ``  |                     
+| Transactions by block | `/txs/?block={hash}`  |   
+| Transactions by address | `/txs/?address={address}`  |   
+| Transactions for multiple addresses | `/addrs/{addr0, addr1, ...}/txs (params: from, to)`  |   
+| POST-variant for ^^ | `(includes params: noAsm, noScriptSig, noSpent)`  |  
```

Comment:
here is an example of the parameters:
`{"addrs":"Dca7Vsv42RAJC6cEdw7dyhLER8QJCHiUYNL,DsTHdW8erZsD1Fqt8uCfapVR1Qsf9KaMwBN","from":3, "to": "5"}`

`{"addrs":"Dca7Vsv42RAJC6cEdw7dyhLER8QJCHiUYNL,DsTHdW8erZsD1Fqt8uCfapVR1Qsf9KaMwBN", "from":3,"to":"5", "noScriptSig":1}`

Only the addrs part is required.  `from` , `to` , `noScriptSig` and `noSpent` are all optional.  Not sure how best to cram that in.